rest-api-spec/src/main/resources/rest-api-spec/test/indices.stats/60_include_both_time_and_gettime_in_stats.yml

```diff
@@ -0,0 +1,32 @@
+---
+setup:
+  - do:
+      indices.create:
+        index: test1
+        body:
+          settings:
+            number_of_shards: 1
+            number_of_replicas: 0
+        wait_for_active_shards: all
+
+  - do:
+      index:
+        index: test1
+        id: 1
+        body: { "foo": "bar" }
+
+  - do:
+      indices.refresh:
+        index: test1
+
+---
+"Test _stats API includes both time and getTime metrics with human filter":
+  - do:
```

Comment:
@hye-on this is great, there are some tests failing, we need to add this section (just before `do` section) to exclude older OS versions (where this change is not present):

```
  - skip:
      version:  " - 2.19.99"
      reason:   "this change is added in 3.0.0"
```